using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -22,64 +25,73 @@ public static void syncIdentifier(final String identifierType,
                                       final VisitorID.AuthenticationState authenticationState);
 ```
 
+o  *identifierType* (String) containing identifier type, and the *identifier* (String) containing identifier value; should not be null or empty. If the identifierTtype and identifier do not have valid string values \(non null or empty\), that identifier is ignored by the Identity extension.
+
+o  *authenticationState* indicates the authentication state of the user, which contains one of the VisitorID.AuthenticationState values `VisitorID.AuthenticationState.AUTHENTICATED`, `VisitorID.AuthenticationState.LOGGED_OUT`, `VisitorID.AuthenticationState.UNKNOWN`
```

Comment:
- Please replace "VisitorID.AuthenticationState" with `VisitorID.AuthenticationState`.

- Please change the text to:
"*authenticationState* indicates the authentication state of the user, which contains one of the following values:

- Please remove the commas between the values and list each of them in a bulleted list like this:
* `VisitorID.AuthenticationState.AUTHENTICATED`
* `VisitorID.AuthenticationState.LOGGED_OUT`
*`VisitorID.AuthenticationState.UNKNOWN`